ReleaseProcedure.txt

```diff
@@ -1,44 +1,52 @@
 These are the tasks typically needed to create an official NHibernate release.
 
-
   * In GitHub, make sure no open issues have milestone set to the release
-    that is about to happen (as these will show in the release notes).
```

Comment:
> as these will show in the release notes

No more true with GitReleaseManager. It only takers into account closed PR/issues.